package.json

```diff
@@ -24,7 +21,6 @@
     "@edgeandnode/eslint-config": "^2.0.3",
     "eslint": "^8.57.0",
     "eslint-plugin-mdx": "^2.3.4",
-    "husky": "^9.0.11",
     "lint-staged": "^15.2.2",
```

Comment:
Should we also remove this?

Reply:
good catch https://github.com/graphprotocol/docs/pull/661/commits/8d2b8ce729c43f9498f720e7e078b60a3e070264